consensus/state_sync/helpers.go

```diff
@@ -6,32 +6,25 @@ import (
 	"google.golang.org/protobuf/types/known/anypb"
 )
 
-func (m *stateSync) SendStateSyncMessage(stateSyncMsg *typesCons.StateSyncMessage, peerAddress cryptoPocket.Address, height uint64) error {
-	anyMsg, err := anypb.New(stateSyncMsg)
+// SendStateSyncMessage sends a state sync message after converting to any proto, to the given peer
+func (m *stateSync) SendStateSyncMessage(msg *typesCons.StateSyncMessage, dst cryptoPocket.Address, height uint64) error {
```

Comment:
Does the helper need to be publically exposed?

Reply:
It is used in `debugging.go`, that is why I made it public. But following your comments and tips (thanks), I will not expose it.